src/slowmokit/methods/metrics/accuracy.cpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file methods/metrics/accuracy.cpp
+ *
+ * Implementation of the Accuracy score main program
+ */
+#include "accuracy.hpp"
+template <class T>
+double accuracy(const std::vector<T> &pred, const std::vector<T> &true_labels)
```

Comment:
Variable names should be in camel case according to our conventions

Reply:
Sir I have done the changes, is it visible to you or should I open pull request again?